icons/drumstick.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
Is this supposed to be called a "drumstick"? @Andreto

Reply:
I think that's what it's called but of i have a better name it then we could change it.